### DISCUSSION TITLE
Want to add our domain to MS white list

### DISCUSSION THREAD

**Discussion by Pjaijai** [Pjaijai/Referalah#66] [討論 | General]
Hi all! I am doing some research on adding our domain to MS white list so that they don't classify us as spam or bad things.

What I found is that I need to join Office 365 to white list it. However, it is a paid service. Any other way to do it ?

(少窮)

**Comment by 14104647ls3** [comment]
@Pjaijai just to be clear, if whitelisted to MS, that means the email will not be marked as spam only in MS domain or other domain as well? Say gmail.

**Reply by Pjaijai** [reply]
Yes correct. I already added our domain to gmail but MS is classifying our domain spam.

**Reply by yaucp** [reply]
maybe just use oauth login then....

**Reply by Pjaijai** [reply]
@yaucp  Not just happen in auth, also happen in contact request email

**Reply by 14104647ls3** [reply]
So I assume what you need right now is start an o365 account for business so that you can register the domain? @Pjaijai

**Reply by 14104647ls3** [reply]
I see Microsoft 365 Business Basic (Nonprofit Staff Pricing) is free, possible to be a temporary solution? [Link](https://www.microsoft.com/en-us/microsoft-365/enterprise/nonprofit-plans-and-pricing)

**Reply by Pjaijai** [reply]
Cool will take a look of it

**Reply by Pjaijai** [reply]
@14104647ls3 @yaucp  Applied, Waiting to MS's approval.
